src/App.js

```diff
@@ -0,0 +1,109 @@
+import { Component } from 'react';
+import AppHeader from './AppHeader';
+import ItemsList from './ItemsList';
+import AppFooter from './AppFooter';
+
+class App extends Component {
+    constructor() {
+        super()
+        this.state = {
+            inputfield: '',
```

Comment:
2 things:
1. Please follow one naming convention, here you have `inputfield` (all lower case characters) and below you have `saveButtonClassActive` (camelCase).
2. Please, try to avoid referring DOM in the state names. So instead of `inputfield`, `newToDoItem` would be much more preferred.

Reply:
Ok, I changed `inputfield` to `newToDoItem`